packages/salesforcedx-vscode-core/test/commands/forceOrgCreate.test.ts

```diff
@@ -18,45 +18,99 @@ import { nls } from '../../src/messages';
 // tslint:disable:no-unused-expression
 describe('Force Org Create', () => {
   describe('Alias Gatherer', () => {
+    const EVENT_CANCEL = 'CANCEL';
+    const EVENT_CONTINUE = 'CONTINUE';
     const TEST_ALIAS = 'testAlias';
     const TEST_WORKSPACE = 'sfdxsimple'; // FYI: This test uses the workspace created by the system tests to run
+    const TEST_ORG_EXPIRATION_DAYS = '7';
+    const TEST_ORG_EXPIRATION_DAYS_PLUS_ONE_DAY = '8';
+    const TEST_ORG_EXPIRATION_DAYS_INPUT_FLOAT = '8.2';
+    const TEST_ORG_EXPIRATION_DAYS_INPUT_INVALID_RANGE = '31';
     let inputBoxSpy: sinon.SinonStub;
 
-    before(() => {
+    beforeEach(() => {
```

Comment:
The before/after were changed to beforeEach and afterEach. There were several reasons for this, the least of which was the running total on the inputBoxSpy number of calls should not be across scenarios. Similarly, the returns for the inputBoxSpy calls should be in the individual scenarios which makes them both easier to read and diagnose if there's a failure.